### ISSUE TITLE
Plugin not working

### ISSUE THREAD

**Issue by Richie76** [Birkbjo/OctoPrint-Themeify#3]
Hi,

I though I could finally get rid of the screaming white of octoprint (thats the only complaint I have about this great software) with your plugin. But there is nothing to control anything - no button, no menu, no nothing.

Only thing I can do is activate or deactivate it.

Color is set to standard - background transparency is off.

Please advise, thanks Richard

**Comment by Birkbjo** [comment]
As if right now it's correct that there is no configuration for the plugin. 

It should be able to just install and restart and it should be enables.

**Comment by Birkbjo** [comment]
Did you get it to work?

**Comment by Richie76** [comment]
Hi,

Nope still not working. It is installed but nothing more.

Can you tell me where I should see a change to access the plugin ? (which screen/menu etc.) ?

Thanks, richard

**Comment by Shuruppak** [comment]
In Firefox Quantum (V57.0) it doesn´t work.
With Microsoft Edge it works fine.

**Comment by Richie76** [comment]
I'm using chrome - should this work?

Am 19.11.2017 11:50 schrieb "Shuruppak" <notifications@github.com>:

In Firefox Quantum (V57.0) it doesn´t work.
With Microsoft Edge it works fine.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Birkbjo/OctoPrint-Themeify/issues/3#issuecomment-345507653>,
or mute the thread

**Comment by Birkbjo** [comment]
Yes, it should just be able to enable it and restart. Have you tried restarting the octoprint server?

Do you have any other plugins?

**Comment by Richie76** [comment]
It is enabled and octoprint was restarted - it asked for it.

Can you show me a screenshot of how it should look - or where I should be
able to see the configuration?

Am 19.11.2017 15:30 schrieb "Birk Johansson" <notifications@github.com>:

Yes, it should just be able to enable it and restart. Have you tried
restarting the octoprint server?

Do you have any other plugins?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Birkbjo/OctoPrint-Themeify/issues/3#issuecomment-345520936>,
or mute the thread

**Comment by Birkbjo** [comment]
The configuration is not yet added, however I'm closing in for another release soon which will include some basic configuration.

After you have restarted it should look like this:
<img width="1175" alt="screen shot 2017-11-19 at 15 41 54" src="https://user-images.githubusercontent.com/13852438/32991679-56108766-cd40-11e7-82dc-abf6ef38ecad.png">

And be enabled in the plugin manager: 
<img width="1173" alt="screen shot 2017-11-19 at 15 43 15" src="https://user-images.githubusercontent.com/13852438/32991681-63355a2a-cd40-11e7-9c07-ee39da9eb170.png">

**Comment by Birkbjo** [comment]
@Shuruppak It works fine for me in Firefox Quantum, what issues do you have?

**Comment by Birkbjo** [comment]
A new version is released which supports customization, can you try and see if it works now?

**Comment by Richie76** [comment]
Sorry for not responding in time. Yesterday out of nowhere my octoprint was
dark themed without any interaction from my side. I think there was an
update on another plugin before - then I turned it off and when I booted
out again it was dark. Like it should have been in the first place.
Thanks for this awesome plugin.

If there is any logfile you wanna see just let me know.

Cheers, Richard

Am 22.11.2017 12:47 vorm. schrieb "Birk Johansson" <notifications@github.com
